cfme/infrastructure/host.py

```diff
@@ -327,7 +327,8 @@ def get_datastores(self):
         datastores = set([])
         for page in paginator.pages():
             for title in sel.elements(
-                    "//div[@id='quadicon']/../../../tr/td/a[contains(@href,'storage/show')]"):
+                    "//div[contains(@id, 'quadicon')]/../../../tr/td/"
+                    "a[contains(@href,'storage/show')]"):
```

Comment:
There currently isn't a title in 5.7.0, but I'll test it locally.
http://pastebin.test.redhat.com/410172